be/src/Project/Services/Content/Content.Api/src/components/blocks/notification-bar-item.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
**suggestion:** Add required attribute for icon or add default option like PushPin. When you set default option, you can ignore my suggestion with constants in notification bar component

Reply:
This resolution doesn't resolve problem when user set wrong name of icon. Bad name of icon set in CMS causes error of the entire website.